sdk/core/azure-core/src/main/java/com/azure/core/annotation/Fluent.java

```diff
@@ -14,8 +14,8 @@
  * annotation, checks can be made to ensure all API meets this expectation. Similarly, classes that are not annotated
  * with this annotation should not have fluent APIs.
  */
-@Target({TYPE})
 @Retention(RetentionPolicy.SOURCE)
+@Target({TYPE})
```

Comment:
To be consistent, either static import `TYPE` or use the class as qualifier everywhere.

```
@Retention(RetentionPolicy.SOURCE)
@Target(ElementType.TYPE)
```

or

```
@Retention(SOURCE)
@Target(TYPE)
```